### DISCUSSION TITLE
Impossible to Write Unit Test with a Component that includes Select

### DISCUSSION THREAD

**Discussion by Stewartarmbrecht** [tamagui/tamagui#2800] [General]
### Current Behavior

I can't find any examples of how to interact with a Select in a jest test that uses @testing-library/react-native.

### Expected Behavior

You can find the Select by a testID and then raise the onValueChange event or something similar.  I tried adding testID to lower level components and using press events but they don't fire the onValueChange event in a jest test.

### Tamagui Version

```markdown
^1.101.3
```

### Platform (Web, iOS, Android)

```markdown
Web
```

### Reproduction

```markdown
Try to write a jest unit test that uses @testing-library/react-native rendor method on a component that has a select.
```

**Comment by Stewartarmbrecht** [comment]
@natew  Why was this converted to a discussion instead of an issue?

**Reply by natew** [reply]
Needs a clear repro!

**Reply by Stewartarmbrecht** [reply]
@natew Sorry I didn't think one was needed given it's primarily an API design issue. I created one. I added two tests that I do not know how to code given the Select API.

https://github.com/Stewartarmbrecht/tamagui-select-test-repro

yarn install
yarn test

See /app_test/(tabs)/two.test.tsx.

**Reply by JayYoungman** [reply]
any update on this, its causing us issues as well!

**Reply by JayYoungman** [reply]
Same issue on web it seems, I can't even assert in a convenient way, when the select dropdown, or sheet is visible after clicking the select trigger with user event, testing-library/react.

EDIT: it seems when selecting the select trigger, from a test, the floating-ui still has pointer events: 'none'. Therefore unable to invoke a user action on a select item.
